src/frontends/paddle/src/op/tile.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs tile(const NodeContext& node) {
+    auto x = node.get_input("X");
+    Output<Node> repeats;
+    if (node.has_input("RepeatTimes")) {
```

Comment:
There looks more than 2 cases of inputs or attributes related to "repeat times". And in the attribute case, the size looks possibly smaller than size of input shape.
 https://github.com/PaddlePaddle/Paddle2ONNX/blob/3ed3454e0e239159129b462bca7e1a229a812647/paddle2onnx/mapper/tensor/tile.cc#L27

Reply:
Sorry, I overlooked the case where there are tensors in the list. This part of the problem has now been completed and unit tests have been added